machine-learning/Dockerfile

```diff
@@ -70,10 +108,20 @@ COPY --from=builder-armnn \
     /opt/ann/build.sh \
     /opt/armnn/
 
+FROM python:3.10-slim-bookworm as prod-rocm
+
+# This is the heavy part of the image, 30Gb of data
+COPY --from=builder-whl-rocm /opt/rocm-6.1.2 /opt/rocm
```

Comment:
Instead of copying all of `/opt/rocm` from the builder, try using `rocm/dev-ubuntu-22.04:6.1.2` as the image.